models/application.js

```diff
@@ -2,6 +2,10 @@
 
 module.exports = function(sequelize, DataTypes) {
     return sequelize.define('application', {
+        form_id: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
```

Comment:
(I've pre-emptively added this field manually to the existing dev/prod databases so we can merge this without issues)